services/app/src/components/team/team.css

```diff
@@ -32,6 +32,24 @@
   flex-direction: column;
   padding: 0px 24px;
   flex-grow: 1;
+  max-height: 392px;
+  overflow:hidden;
+  overflow-y:auto;
+}
+
+.team-list-items::-webkit-scrollbar {
```

Comment:
is this scrollbar styling applies only for `team-list-items` or it should be consistent across the application?